index.html

```diff
@@ -30,7 +30,6 @@ <h1>Hello world ,we are software devlopers!</h1>
           <li></li>
           <li></li>
           <li></li>
-          <li></li>
```

Comment:
-Please use lists that are required for your project only. It doesn't make sense to empty list bullets